bin/cheritest/cheritest_vm.c

```diff
@@ -312,6 +313,19 @@ xfail_need_writable_tmp(const char *name __unused)
 	return (reason);
 }
 
+const char* xfail_need_writable_non_tmpfs_tmp(const char *name)
+{
+	struct statfs info;
+	if (statfs("/tmp", &info) != 0) {
+		cheritest_failure_err("statfs /tmp");
+	}
+	if (strcmp(info.f_fstypename, "tmpfs") == 0) {
+		return ("cheritest_vm_notag_tmpfile_shared needs non-tmpfs /tmp");
+	}
+	/* /tmp is not tmpfs, now check that it is writable */
```

Comment:
style(9) would like a blank line before block comments, and for block comments to be complete sentences including suitable uppercasing and punctuation.